src/main/java/org/apache/sling/feature/io/artifacts/ArtifactHandler.java

```diff
@@ -46,11 +47,24 @@ public ArtifactHandler(final String url, final URL localURL) {
      * @param file The file for the artifact
      * @throws MalformedURLException If the file name cannot be converted to a URL.
      * @since 1.1.0
+     * @deprecated Use {@link #ArtifactHandler(Path)} instead
      */
+    @Deprecated
```

Comment:
Why should we deprecate methods using File? Today java.io.File is not deprecated and used everywhere, so it seems convenient to have this

Reply:
I agree, fixed in https://github.com/apache/sling-org-apache-sling-feature/pull/26/commits/fafa5e407baeed8ec0f7113951f8c0c8f6d37d7f.